kafka/client_async.py

```diff
@@ -526,6 +526,8 @@ def poll(self, timeout_ms=None, future=None, delayed_tasks=True):
             timeout_ms = 100
         elif timeout_ms is None:
             timeout_ms = self.config['request_timeout_ms']
+        elif not isinstance(timeout_ms, int) and not isinstance(timeout_ms, float):
+            raise RuntimeError('Invalid type for timeout: %s' % (type(timeout_ms),))
```

Comment:
The argument to `%` doesn't have to be a tuple if it's just a single argument:

    >>> 'Invalid type for timeout: %s' % (type(5),)
    "Invalid type for timeout: <type 'int'>"
    >>> 'Invalid type for timeout: %s' % type(5)
    "Invalid type for timeout: <type 'int'>"

I'd argue that in this case the latter form is more readable.

Reply:
Yes, I know. But I usually prefer to make it explicit as a tuple to avoid ambiguity with the case where the argument is a tuple. It's not the case here, since we know that `type(5)` will never be a tuple. But I usually try to use the same style for consistency.

I'm happy to change this since the style for this project is the latter form.